cluster.yml

```diff
@@ -30,15 +30,24 @@
     - { role: docker, tags: docker }
     - { role: rkt, tags: rkt, when: "'rkt' in [ etcd_deployment_type, kubelet_deployment_type ]" }
 
-- hosts: etcd:!k8s-cluster
+- hosts: all
```

Comment:
Yep, ran into issues on my end as well with this. I've changed it back in the new set of changes coming up.